templates/sidebar.html.twig

```diff
@@ -93,6 +93,31 @@
                         </div>
                     </div>
                 </li>
+                <li class="{% if app.request.get('overlay') %}mx-2{% else %}mb-6{% endif %}">
+                {% if is_granted('IS_AUTHENTICATED_REMEMBERED') %}
```

Comment:
Are you sure that "_REMEMBERED" will not exclude some users and hide the button for them ?

Reply:
It's the same permission for stream tit:e update, if we change one we should change both

Reply:
Sorry, i was wrong. According to https://symfony.com/doc/current/security.html#checking-to-see-if-a-user-is-logged-in-is-authenticated-fully

I thought that there was an "IS_AUTHENTICATED" role.

IS_AUTHENTICATED_REMEMBERED: All logged in users have this, even if they are logged in because of a "remember me cookie". Even if you don't use the remember me functionality, you can use this to check if the user is logged in.